finat/fiat_elements.py

```diff
@@ -275,7 +275,8 @@ def is_const(expr):
             assert table.shape[-1] == m
             zerocols = np.isclose(abs(table).max(axis=tuple(range(table.ndim - 1))), 0.0)
             if all(np.logical_or(const_mask, zerocols)):
-                vals = base_values_sympy[const_mask]
+                # Casting is safe by assertion of is_const
```

Comment:
I don't see such an assertion anywhere. Did you mean to add it and forget?

Reply:
The `is_const` function does it. It tries to cast the contents to `float` and sets `const_mask` to True in those cases, otherwise it sets them to `float`

Reply:
got it